src/test/java/teammates/test/cases/BaseComponentTestCase.java

```diff
@@ -106,9 +106,21 @@ protected static void verifyPresentInDatastore(FeedbackQuestionAttributes expect
         if (wildcardId) {
             expected.setId(actual.getId());
         }
+        
+        setEqualTimestampBetweenExpectedAndActual(expected, actual);
+        
         assertEquals(gson.toJson(expected), gson.toJson(actual));
     }
     
+    private static void setEqualTimestampBetweenExpectedAndActual(FeedbackQuestionAttributes expected,
+                                    FeedbackQuestionAttributes actual) {
+        
+        //timestamps are ignored
```

Comment:
May not be as important, but proper English (capitalization, etc) and a space after `//` will definitely improve the readability of comments.